src/tools/check_brokenURL.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
+
+import re
+import os
+import sys
+import getopt
+import requests
+import argparse
+
+def get_filepaths(directory):
+    file_paths = []  
+    for root, directories, files in os.walk(directory):
+        for filename in files:
+            filepath = os.path.join(root, filename)
+            file_paths.append(filepath)
+    return file_paths
+
+def checkBrokenLink(url):
+    if requests.get(url).ok == True:
```

Comment:
@SpandanKumarSahu 
` I believe if it is being redirected, then it should still be a broken link.`

i think HTTP301/302 is okay, user could visit link via browser.